src/accountsdb/snapshots.zig

```diff
@@ -1452,22 +1465,22 @@ pub const AccountsDbFields = struct {
 /// contains all the metadata from a snapshot.
 /// this includes fields for accounts-db and the bank of the snapshots slots.
 /// this does not include account-specific data.
-pub const SnapshotFields = struct {
+pub const Manifest = struct {
     bank_fields: BankFields,
     accounts_db_fields: AccountsDbFields,
     /// incremental snapshot fields.
     bank_extra: ExtraFields,
 
-    pub fn deinit(self: SnapshotFields, allocator: std.mem.Allocator) void {
-        self.bank_fields.deinit(allocator);
-        self.accounts_db_fields.deinit(allocator);
-        self.bank_extra.deinit(allocator);
+    pub fn deinit(man: Manifest, allocator: std.mem.Allocator) void {
```

Comment:
lets keep consistent with the self pattern

Reply:
6d69637973392273c88393596149d8558455e0d1